src/main/resources/templates/orderLookup.html

```diff
@@ -3,16 +3,14 @@
 <head>
     <meta charset="UTF-8">
     <title>Order Lookup</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
```

Comment:
I included Bootstrap here so I could include some basic CSS classes in the page.